### PR TITLE
Get articles about particle (Create providers directory for CERN)

### PR DESCRIPTION
**Pull request by DanielSBrown**
🎶  No good migrations. Just sweet harvestations 🎶 

**Changes**

- Added providers folder for Switzerland TLD (ch).

- Added folder for CERN.

- No migrations folder, but harvesting generates the appropriate JSON files.

- CERN is OAI (cdsweb.cern.ch/oai2d/).

**Side Effects**

- Arxiv already harvests some CERN publications.

### CODE REVIEW

**Comment by leb2dg** [comment]
After running `./manage.py makeprovidermigrations` this goes through the whole pipeline 👍

File: providers/ch/cern/__init__.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
+default_app_config = 'providers.ch.cern.apps.AppConfig'
+
+"""<record>
```
Comment (leb2dg) [inline providers/ch/cern/__init__.py]: Can you format this like:
```
"""
Example Record

<record>
...
```

**Comment by chrisseto** [comment]
Please add the migration to this PR

**Comment by DanielSBrown** [comment]
@chrisseto requested changes made.

**Comment by chrisseto** [comment]
@DanielSBrown Sorry, only add the migration for cern.
`git reset HEAD~1 --soft` is you friend

**Comment by DanielSBrown** [comment]
How about now @chrisseto

**Pull request merged**